app/models/user.rb

```diff
@@ -47,6 +47,27 @@ class User
     validates :username, presence: true
   end
 
+  def self.valid_google_domain?(email)
+    return true if Errbit::Config.google_authorized_domains.nil?
+    match_data = /.+@(?<domain>.+)/.match(email)
```

Comment:
This is probably fine since the `@` anchors the position, but I think it would be better to ensure this is the end of the string `/.+@(?<domain>.+)$/`